arkfilepicker/src/main/java/space/taran/arkfilepicker/ArkFilePickerConfig.kt

```diff
@@ -3,6 +3,7 @@ package space.taran.arkfilepicker
 import androidx.annotation.PluralsRes
 import androidx.annotation.StringRes
 import androidx.annotation.StyleRes
+import space.taran.arkfilepicker.presentation.filepicker.ArkFilePickerMode
 import java.nio.file.Path
 
 class ArkFilePickerConfig(
```

Comment:
I just looked at `ArkFilePickerConfig` usages in Navigator.. Do we invoke it only during resources copy/move? Don't we invoke it when we want to add new folder in Folders screen?

Reply:
Right now the file picker is only used to copy/move in Navigator. Folder selection requires additional configuration of the pick button, such as root/favorites/already added

Reply:
Actually, we don't need this mode for adding a folder as root/favorite. It would be too complicated to present only ARK folders on top level, but switching to filesystem mode when any of them is clicked.